client/state/sites/plans/actions.js

```diff
@@ -151,3 +152,17 @@ export function refreshSitePlans( siteId ) {
 		dispatch( fetchSitePlans( siteId ) );
 	};
 }
+
+/**
+ * Returns an action object to be used in signalling that site plan ownership
+ * change to another user has started.
+ *
+ * @param {Number} siteId - ID of the site
+ * @param {Number} newUserId - ID of the new user
+ * @returns {Object} the corresponding action object
+ */
+export const transferPlanOwnership = ( siteId, newUserId ) => ( {
+	type: SITE_PLAN_OWNERSHIP_TRANSFER,
+	newUserId,
```

Comment:
NIT: Not really a new user… maybe `newOwnerUserId`?

Reply:
Good point, updated in https://github.com/Automattic/wp-calypso/pull/25409/commits/80f099cb5da8a375a091a99f454fc5e4706e53dc